src/index.js

```diff
@@ -495,6 +495,45 @@ export default class HyperScreen extends React.Component {
       this.onUpdateFragment(href, action, currentElement, opts);
     } else if (action === ACTIONS.SWAP) {
       this.onSwap(currentElement, opts.newElement);
+    } else if (action === ACTIONS.DISPATCH_EVENT) {
+      const { behaviorElement } = opts;
+      const eventName = behaviorElement.getAttribute('event-name');
+      const trigger = behaviorElement.getAttribute('trigger');
+      const ranOnce = behaviorElement.getAttribute('ran-once');
+      const once = behaviorElement.getAttribute('once');
+      const delay = behaviorElement.getAttribute('delay');
```

Comment:
That's how it's done in the entire codebase. Shall I change it specifically here?

Reply:
Oh really? Are they pre-parsed at a different layer?

Reply:
I think I misunderstood it since other places are using a `later` function which does the parsing. Added a `parseInt` here since I'm doing a `setTimeout`. Also switched to using a serialized `'true'` instead of a boolean `true`.